client/flutter/lib/api/content_loading.dart

```diff
@@ -85,12 +89,14 @@ class ContentLoading {
   /// Load a localized content bundle from the network, throwing an exception if not found.
   Future<ContentBundle> _loadFromNetwork(String name, String suffix) async {
     var url = '$baseContentURL/${_fileName(name, suffix)}';
-    var response = await http.get(url,
-        headers: {"Accept": "application/yaml"}).timeout(networkTimeout);
-    if (response.statusCode != 200) {
-      throw Exception("Error status code: ${response.statusCode}");
+    var headers = {"Accept": "application/yaml"};
+    File file = await WhoCacheManager()
+        .getSingleFile(url, headers: headers)
+        .timeout(networkTimeout);
+    if (file == null) {
+      throw Exception("File not retrieved from network or cache: $url");
     }
-    return ContentBundle.fromBytes(response.bodyBytes);
+    return ContentBundle.fromBytes(file.readAsBytesSync());
```

Comment:
this sync call - does it hold up the ui thread ?

Reply:
No, those are both called within async load methods (and they are also very fast anyway).

Reply:
I don't think this is how async works.  Only if you use await do you allow the main event loop to keep running. 

Perhaps:
```suggestion
    return ContentBundle.fromBytes(await file.readAsBytes());
```

Reply:
Ah, sorry, I didn't even see what you highlighted (the sync method).  Yes, that is better... Although I don't think it would be a problem here.  Adding your suggestion!